beetmoverscript/script.py

```diff
@@ -43,14 +46,18 @@
     #   a. download artifact
     #   b. upload to candidates/dated location
     await move_beets(context, manifest)
-
+    # 6. write balrog_manifest to a file and add it to list of artifacts
+    if context.task["payload"]["update_manifest"]:
```

Comment:
hm, er, what am I talking about.

in the payload we could revert back to have: context.task["payload"]["template_key"] which just states the explicit template key.

then, yes, at this line we go with what you suggest `if context.balrog_manifest:`